packages/keystone/runtimeStats.js

```diff
@@ -103,20 +103,27 @@ class RuntimeStatsMiddleware {
                 logger.info({
                     msg: 'current values',
                     runtimeStats: otherStats,
+                    activeRequestsCount: activeRequestsIds.size,
```

Comment:
It's better to keep all stats under the same path
```suggestion
                    runtimeStats: { ...otherStats, activeRequestsCount: activeRequestsIds.size },
```